hotkey.js

```diff
@@ -14,12 +16,56 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function choseTheElement(list, current) {
+  let arr = [...list];
+  
+  for (let j = 0; j < arr.length;) {
+    let element = arr[j];
+    
+    while (true) {
```

Comment:
Too complex logic. There is a simpler way to write it. For instance, using `:not([data-keyux-ignore-hotkeys]) [aria-keyshortcuts=…]`